docs/ECLLanguageReference/ECLR_mods/BltInFunc-JOIN.xml

```diff
@@ -646,6 +646,13 @@
 
     <para>Use SMART when the right side dataset is likely to be small enough
     to fit in memory, but is not guaranteed to fit.</para>
+
+    <para>If your job is spilling to disk or you get this error:</para>
```

Comment:
Whether your job is spilling to disk or not, isn't relevant.

Lookup join must fit the entire RHS (global RHS) in memory in every slave.
If it can not, there will me a memory exhausted error.